.travis.yml

```diff
@@ -27,6 +28,9 @@ addons:
     - travis-ci/worker/$(git describe --always --dirty --tags)
     - travis-ci/worker/$TRAVIS_BRANCH
 
+before_install:
+- go get -u "${SHFMT_URL}" ;
```

Comment:
I'm 👎 to asserting bleeding edge `shfmt` via CI.  I would rather pin to a released version.  I see that the tests are failing because this installation of bleeding edge `shfmt` is superseded by another installation of `v1.3.1` below.  I would like to propose that both of these be removed and instead install the binary that's provided by the project a la:

``` yaml
- curl -o ~/bin/shfmt -sSL https://github.com/mvdan/sh/releases/download/v2.0.0/shfmt_v2.0.0_linux_amd64                                                                                       
- chmod +x ~/bin/shfmt                                                                                                                                                                         
- shfmt -version
```

Incidentally (!) this is what I'm doing in another branch of this repo :grin: https://github.com/travis-ci/worker/blob/e8a7520e10d458b6f8857b13d13fe57a1932a023/.travis.yml#L34-L36